src/main/java/world/bentobox/bentobox/listeners/flags/protection/BlockInteractionListener.java

```diff
@@ -31,31 +30,12 @@
 public class BlockInteractionListener extends FlagListener
 {
 
-    /**
-     * These cover materials in another server version. This avoids run time errors due to unknown enum values, at the
-     * expense of a string comparison
-     */
-    private static final Map<String, String> stringFlags;
-    private static final String CHEST = "CHEST";
-
-    static
-    {
-        stringFlags = Map.of(
-                "ACACIA_CHEST_BOAT", CHEST,
-                "BIRCH_CHEST_BOAT", CHEST,
-                "JUNGLE_CHEST_BOAT", CHEST,
-                "DARK_OAK_CHEST_BOAT", CHEST,
-                "MANGROVE_CHEST_BOAT", CHEST,
-                "OAK_CHEST_BOAT", CHEST,
-                "SPRUCE_CHEST_BOAT", CHEST);
-    }
-
     /**
      * Handle interaction with blocks
      *
      * @param e - event
      */
-    @EventHandler(priority = EventPriority.LOWEST, ignoreCancelled = true)
+    @EventHandler(priority = EventPriority.NORMAL, ignoreCancelled = true)
```

Comment:
Changed priority to enable sign shops to be used. Previously it had no affect because we never checked signs, but now that we do, the priority needs to be higher. ChestShop uses LOW.